addon/components/base/bs-form/element.js

```diff
@@ -330,7 +330,7 @@ const nonDefaultLayouts = A([
  */
 export default FormGroup.extend({
   layout,
-  classNameBindings: ['disabled:is-disabled', 'required:is-required', 'isValidating'],
+  classNameBindings: ['disabled:disabled', 'required:is-required', 'isValidating'],
```

Comment:
Actually, if BS4 supports the `disabled` class as it seems (I have seen `disabled` for BS3 only for `input` and `fieldset`), we should support it! The `is-desiabled` class was never documented, so should be ok to change that, at least while still in beta. So let's keep `disabled`!